e2e/suites/actions-available/files-folders/generic.test.ts

```diff
@@ -150,13 +149,6 @@ describe('Generic tests : ', () => {
       await page.goToFavoriteLibraries();
       expect(await toolbar.isEmpty()).toBe(true, `actions displayed though nothing selected`);
     });
-
-    it('[C291815] on Search Results', async () => {
```

Comment:
Why are you removing this test?

Reply:
This test is not valid anymore. (toggle search filter action has been removed)

Reply:
The test is still valid, check the name of the describe.
Just that now you need another way of checking this.
Might work with toolbar.isEmpty() to be true